organization/_posts/2021/2021-11-25-coordination.md

```diff
@@ -0,0 +1,168 @@
+
+It was felt to useful to submit a paper to this, so Graeme, Liz and Eduardo wrote/contributed to [what we submitted](https://docs.google.com/document/d/1f3wcOgE8tqh-Q88A3wzQrZaWhLazPl196g1lLkv0XZo/edit?usp=sharing).
+
+- We seem to be more of an "interesting problem" than a solution in this space, but no bad idea to throw our hat into the ring.*
+
+## Working Group Updates
+
+### Mandates for 2022
+
+We have started to think about WG convenerships for 2022. For WG convenors, please let Benedikt know your plans.
+
+- Normally WG conveners serve for 2 years, by mutual agreement (but this is not set in stone).
+- Most conveners want to continue; some would make space for younger replacements if there are any.
+
+Outcome: we anticipate a call for WG conveners in 7 of the 8 groups (the exception is Simulation where everyone stays on). The search committee would by default be Graeme, Liz, Michel and Benedikt.
+
+Email will be sent on Monday with a call for nominations and a 2 week window.
+
+### Data Analysis
+
+- Metadata document now in [overleaf](https://www.overleaf.com/read/cxkkdqdrrfcf), to be submitted to CSBS. Need ORCIDs and institute info from people still (probably searchable)
+- Will have meeting before Xmas on workflow management tools - potential benchmarks here
+  - Have spoken to 'Luigi' developers, planning to find (LHCb?) speaker on SnakeMake and possibly also CWL
+- Start of 2022, may arrange a meeting on representing/book-keeping systematic uncertainty variations
+- Will there be a HSF workshop in the Spring?
+  - IRIS-HEP is also thinking about this, and due to the CHEP cancellation, the corresponding dates (May) may be an appropriate time
+  - People in HSF seem generally keen as well
+  - Analysis ecosystem is the preferred topic, practical to keep it constrained (hybrid, ~60-70 attendees)
+    - Aim to put together organising team by end of the year
+    - Gordon can look into option to host in Marseille
+
+### Detector Simulation
+
+- Presentation on December 13th on FPGAs for simulation:
+  - <https://indico.cern.ch/event/1096432/>
+- Now thinking about/planning meetings for 2022
+- Talk on simulation at Snowmass small experiment workshop
+  - <https://indico.physics.lbl.gov/event/1756/contributions/6251/>
+  - Noted that all small experiments rely heavily on Geant4 and need it to be sustained by the FAs
+
+### Reconstruction and Software Triggers
+
+- One outcome of 2021 survey for Reconstruction was the need/request for R&D detector dataset:
+  - Open Data Detector - Tracking System (follow-up of TrackML detector) to be ACTS-free (MR + ACAT poster)
+  - Calorimeter system will be added as part of a bigger initiative of CERN EP-R&D allows full Geant4 simulation of realistic tracking (including timing) + calorimeter
```

Comment:
Not sure how to rephrase the sentence without changing the present meaning, which is a bit fuzzy.

Reply:
I had a go at making it clearer